### DISCUSSION TITLE
RDBMS Resource Usage when Using PrestoDB

### DISCUSSION THREAD

**Discussion by daun-gatal** [prestodb/presto#16179] [Q&A] [answered]
When we querying mysql database using presto, is it means that we still using mysql’s resource like cpu or ram or not?. Thank you

**Comment by bitsondatadev** [comment, marked as answer]
Yes. In the ideal scenario, MySQL will be able to utilize its own data structures and indices to answer the query before handing back the result to Presto. 

The real benefit to using Presto on relational databases is the ability to run queries from a single access point to multiple databases, or in particular, running federated queries (i.e. Joins and Unions) over multiple databases. There is no benefit to running Presto over a single relational database unless you plan to add more connectors in the future. The same doesn't apply to Hive or data lake connectors like Iceberg though.

If this answers the question please accept and check on my profile to get in contact with me if you need any more information or something's still not clear.

**Reply by daun-gatal** [reply]
I see. Thank you for the answer @bitsondatadev
